ui/lib/core/addon/components/confirm-action.hbs

```diff
@@ -35,6 +35,7 @@
           </p>
         </div>
         <div class="confirm-action-options">
+          {{! TODO Hds::Button replacement - skipping because modal will replace this confirm inline-popup menu }}
```

Comment:
This component is deprecated and will be replaced by a modal
<img width="229" alt="Screenshot 2023-11-13 at 6 03 37 PM" src="https://github.com/hashicorp/vault/assets/68122737/ae7808ca-ef9f-4a7d-8881-d54eee337901">